### ISSUE TITLE
segfault with zig-lib-dir

### ISSUE THREAD

**Issue by Pyrolistical** [ziglang/zig#13875]
### Zig Version

0.11.0-dev.672+e94504543

### Steps to Reproduce and Observed Behavior

Run `zig test --zig-lib-dir lib lib/std/enums.zig`

Crashes
```sh
zsh: segmentation fault  zig test --zig-lib-dir lib lib/std/enums.zig
```

### Expected Behavior

Test runs and passes.

**Comment by Pyrolistical** [comment]
I have a really old copy of zig @ 8a5818535 and it doesn't crash.

**Comment by kcbanner** [comment]
If you go to the tarball builder actions (https://github.com/ziglang/www.ziglang.org/actions), you can find the version numbers in the build results (ie. https://github.com/ziglang/www.ziglang.org/actions/runs/3657177569/jobs/6180475220). Then you can download it via `https://ziglang.org/builds/zig-<os>-<arch>-<version>.(tar|xz)`

The oldest one I have locally is `0.11.0-dev.537+36da3000c` and I'm seeing it there as well.

**Comment by Vexu** [comment]
Likely caused by #13821, are you certain your library and Zig executable are the same version?

**Comment by kcbanner** [comment]
That seems correct, with `.672+e94504543` I can build `e4874d842ec3a771a5c438b4c48020a74c221935` which is the one before #13821 was merged.

**Comment by Pyrolistical** [comment]
I see. So this should be resolved once a build after #13821 is released.

**Comment by Pyrolistical** [comment]
its fixed on 0.11.0-dev.681+f02073434
